pkg/build/docker_generic.go

```diff
@@ -23,9 +23,7 @@ type DockerGenericBuilder struct {
 }
 
 type DockerGenericBuilderConfig struct {
-	BuildArgs    map[string]*string `toml:"build_args"` // ok if nil
-	PushRegistry bool               `toml:"push_registry"`
-	RegistryType string             `toml:"registry_type"`
+	BuildArgs map[string]*string `toml:"build_args"` // ok if nil
```

Comment:
We might one day want to support other registries. I never much liked the "registry_type" model. Since they are all docker registries, you just need credentials and the regular docker client.

You can log into ecs with the regular docker client, for example. We could potentially support any docker endpoint by just using the regular docker client and providing a model to pass credentials

Reply:
We actually support DockerHub apart from AWS ECR.